doc/transform_methods/comparison/fig_benchmarks/pyabel-plot.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: UTF-8 -*-
```

Comment:
Is this file actually used any longer?

Reply:
It is not used in the final documentation but is useful for looking quickly at the results while running benchmarks (however, then it needs `benchmark_dir` to point at `'working'`, which I forgot to change). I can either delete it or modify to complement `pyabel-timing.py`. What do you think?

Reply:
Well, I have committed the usable version, just in case, but it can be deleted completely.

Reply:
I think that it's fine to keep it around so that it can be used for looking at the benchmarks. Thanks for fixing it!